### DISCUSSION TITLE
[Projects Beta] Current sprint closed one day earlier than expected

### DISCUSSION THREAD

**Discussion by OscarSal** [community/community#20101] [Projects and Issues] [answered]
In Projects Beta we have detected an issue, our current sprint has closed a day earlier than expected because it is not considering the closing day (July 4, 2022) as part of the sprint and in fact it is, it is the last day that is still part of it.

![image](https://user-images.githubusercontent.com/100857600/177561956-7c28ba8d-b06e-4cee-8bd3-e52fa4d08209.png)

There is no issues in the view we use to display the current iteration within the sprint (filtering by `iteration:@Current`).

**Comment by Franklin89** [comment]
Having the same issue. The Board View returns an empty list if filter by `iteration: @current` but the table view seems to work.

![image](https://user-images.githubusercontent.com/4151467/178414414-e009c380-7454-4419-a412-a46a281fcee0.png)

**Comment by zoebosschaart** [comment]
Good to see that I was not the only one encountering this issue last week! Raised a support-issue but they said they wouldn't do anything with that and that projects-beta issues only can be raised here.

**Comment by 2percentsilk** [comment]
Hey folks! Thanks for filing this - I'm investigating with the team and will share out details as I know more. Looks like we may have a pesky off-by-one error 👀

**Reply by 2percentsilk** [reply]
@OscarSal @zoebosschaart @Franklin89 can you confirm if this is still happening for y'all? We shipped (what we think is) a fix a week or so ago and I can't recreate this issue on my personal or test accounts so looking for any more details about this.

**Reply by zoebosschaart** [reply]
I guess I can only test this upcoming Monday, as that will be the last day of our Iteration. Will come back to you then.

**Reply by zoebosschaart** [reply]
@2percentsilk yesterday we did not encounter any issues, so bugfix is helping!

**Comment by Neudrino** [comment]
There are several issues describing the same problem. Posting this just for reference.

- https://github.com/github-community/community/discussions/20101 This is the current issue and the most up-voted one.
- https://github.com/github-community/community/discussions/20217
- https://github.com/github-community/community/discussions/20032
- https://github.com/github-community/community/discussions/20108
- https://github.com/github-community/community/discussions/18834

**Comment by 2percentsilk** [comment, marked as answer]
Based on testing from the community and team, we believe we have fixed this bug. Thanks to everyone for reporting it!
